docs/USAGE-GUIDE.md

```diff
@@ -607,66 +607,41 @@ In TLS1.3, session ticket messages are sent after the handshake as "post-handsha
 
 Additionally, in TLS1.3, multiple session tickets may be issued for the same connection. Servers can call `s2n_config_set_initial_ticket_count()` to set the number of tickets they want to send and `s2n_connection_add_new_tickets_to_send()` to increase the number of tickets to send during a connection.
 
-### s2n\_config\_set\_client\_hello\_cb
+### Session Resumption Forward Secrecy
 
-```c
-int s2n_config_set_client_hello_cb(struct s2n_config *config, s2n_client_hello_fn client_hello_callback, void *ctx);
-```
+In TLS1.2, the secret stored inside the ticket is the original session's master secret. Because of this, TLS1.2 session tickets are not forward secret, that is, compromising the session ticket key exposes both current and past sessions' encrypted data.
 
```

Comment:
Nit
```suggestion
In TLS1.2, the secret stored inside the ticket is the original session's master secret. Because of this, TLS1.2 session tickets are not forward secret, meaning that compromising the session ticket key exposes both current and past sessions' encrypted data.

```